src/groupeddataframe/grouping.jl

```diff
@@ -365,7 +365,7 @@ function _makeheaders(fs::Vector{T}, cn::Vector{Symbol}) where T<:Function
 end
 
 function _aggregate(d::AbstractDataFrame, fs::Vector{T}, headers::Vector{Symbol}, sort::Bool=false) where T<:Function
-    res = DataFrame(Any[vcat(f(d[i])) for f in fs for i in 1:size(d, 2)], headers)
+    res = DataFrame(Any[vcat(f(d[i])) for f in fs for i in 1:size(d, 2)], headers) # after deprecation period this will disallow duplicate names
```

Comment:
I don't think this kind of comment is useful. On the contrary, we could forget to remove it, and it would be confusing.

Reply:
removed

Reply:
actually I changed it to target comment - OK?

Reply:
Not sure, it's kind of weird to say this here as there is no reason to think there would be duplicate names. Why would the reader care? I'd remove these comments and keep them only in place where there's a special reason to mention this fact.

Reply:
OK